inference/core/roboflow_api.py

```diff
@@ -325,6 +326,29 @@ def get_roboflow_labeling_jobs(
     return _get_from_url(url=api_url)
 
 
+@wrap_roboflow_api_errors()
+def get_deployment_specification(
+    api_key: str,
+    workspace_id: WorkspaceID,
+    deployment_name: str,
+) -> dict:
+    api_url = _add_params_to_url(
+        url=f"{API_BASE_URL}/{workspace_id}/deployments/{deployment_name}",
```

Comment:
do we need to use `inference.core.utils.url_utils.wrap_url` here?

Reply:
as we discussed, it is done down the line